nemo/lightning/io/mixin.py

```diff
@@ -415,6 +415,39 @@ def import_ckpt(self, path: str, overwrite: bool = False, base_path: Optional[Pa
         connector.on_import_ckpt(self)
         return ckpt_path
 
+    @classmethod
+    def import_ckpt(cls, path: str, overwrite: bool = False, base_path: Optional[Path] = None, **kwargs) -> Path:
```

Comment:
how is it different from `https://github.com/NVIDIA/NeMo/blob/main/nemo/lightning/io/api.py#L111`

what's the use case of this?

Reply:
Ah, thanks for the pointer~I'll remove the classmethod